src/primitives.js

```diff
@@ -799,7 +817,7 @@ function createCubeVertices(size) {
  *     truncated cone.
  * @param {boolean} [opt_topCap] Create top cap. Default = true.
  * @param {boolean} [opt_bottomCap] Create bottom cap. Default = true.
- * @return {Object.<string, WebGLBuffer>} The created cone buffers.
+ * @return {module:twgl/primtives.StandardBuffers} The created cone buffers.
```

Comment:
typo (recurring)

Reply:
"primtives" => "primitives"

Reply:
```
export type StandardArrays = {
        position: Float32Array;
        texcoord: Float32Array;
        normal: Float32Array;
        indices: Uint16Array;
        [key: string]: TypedArray;
    };
```
this is nothing to do with the code, because the output of the types will
```
{
        position: Float32Array;
        texcoord: Float32Array;
        normal: Float32Array;
        indices: Uint16Array;
        [key: string]: TypedArray;
} <=> {
        [key: string]: TypedArray;
};
```
it's better to do something like
```
export type StandardArrays<T> = {
        position: Float32Array;
        texcoord: Float32Array;
        normal: Float32Array;
        indices: Uint16Array;
        [keyof T]: TypedArray; // or something like ...T
    };
```
I'd do it myself, but there won't be time for another month.